sdk/nodejs/metadata.ts

```diff
@@ -36,3 +37,17 @@ export function getStack(): string {
     }
     throw new Error("Stack unknown; are you using the Pulumi CLI?");
 }
+
+/**
+ * getStackTag returns a stack tag's value or undefined if it is unset.
+ */
+export function getStackTag(name: string): string | undefined {
+    return runtime.getStackTag(name);
+}
+
+/**
+ * getStackTags returns a copy of all the stack tags.
+ */
+export function getStackTags(): {[name: string]: string} {
```

Comment:
@lukehoban, We didn't do a formal review of these so just want to make sure you're OK with these two new APIs: `getStackTag` and `getStackTags`. There are similarly named APIs for Go and Python.

Reply:
Node:

```typescript
/**
 * getStackTag returns a stack tag's value or undefined if it is unset.
 */
export function getStackTag(name: string): string | undefined {
    return runtime.getStackTag(name);
}

/**
 * getStackTags returns a copy of all the stack tags.
 */
export function getStackTags(): {[name: string]: string} {
    return runtime.allStackTags();
}
```

Python:

```python
def get_stack_tag(name: str) -> str:
    """
    Returns a stack tag's value or None.
    """
    return runtime.get_stack_tag(name)


def get_stack_tags() -> Dict[str, str]:
    """
    Returns a copy of all the stack tags.
    """
    return runtime.get_stack_tags()
```

Go:

```go
// GetStackTag returns the stack tag value, as a string, and a bool indicating whether it exists or not.
func (ctx *Context) GetStackTag(name string) (string, bool) {
	v, ok := ctx.info.StackTags[name]
	return v, ok
}

// GetStackTags returns a copy of the stack tags.
func (ctx *Context) GetStackTags() map[string]string {
	if ctx.info.StackTags == nil {
		return nil
	}

	// Make a copy of the tags.
	copy := make(map[string]string)
	for k, v := range ctx.info.StackTags {
		copy[k] = v
	}
	return copy
}
```